app/views/admin/_menu.html.erb

```diff
@@ -0,0 +1,10 @@
+<li class="dropdown">
+  <%= link_to sufia.admin_stats_path, role: 'button', data: { toggle: 'dropdown' }, aria: { haspopup: true, expanded: false } do %>
+    <span class="fa fa-gear"></span> <%= t("sufia.toolbar.admin.menu") %> <span class="caret"></span>
+  <% end %>
+  <ul class="dropdown-menu">
+    <% if respond_to? :admin_stats_path %>
```

Comment:
Yes, this is now true. Related: we should remove this conditional as well: https://github.com/projecthydra/sufia/blob/master/config/routes.rb#L153

I'd be fine with seeing that snuck into this PR.